das_api/src/builder.rs

```diff
@@ -31,7 +35,22 @@ impl RpcApiBuilder {
         module.register_async_method(
             "get_assets_by_owner",
             |rpc_params, rpc_context| async move {
-                let payload = rpc_params.parse::<GetAssetsByOwner>()?;
+                let payload: GetAssetsByOwner;
+                if let Ok(parsed_payload) = rpc_params.parse::<GetAssetsByOwner>() {
```

Comment:
What's the point in doing the parse? We might as well just use sequence each time, right?